src/wasm.cc

```diff
@@ -198,6 +197,10 @@ void WasmBase::registerCallbacks() {
   } else if (abiVersion() == AbiVersion::ProxyWasm_0_2_0) {
     _REGISTER_PROXY(continue_stream);
     _REGISTER_PROXY(close_stream);
+  } else if (abiVersion() == AbiVersion::ProxyWasm_0_2_1) {
+    _REGISTER_PROXY(continue_stream);
+    _REGISTER_PROXY(close_stream);
+    _REGISTER_PROXY(get_log_level);
   }
 #undef _REGISTER_PROXY
 }
```

Comment:
There are also `_GET_PROXY` callbacks below that are guarded by the ABI version.